src/main/java/org/openstreetmap/atlas/geography/atlas/bloated/BloatedRelation.java

```diff
@@ -241,25 +242,55 @@ public BloatedRelation withIdentifier(final long identifier)
         return this;
     }
 
-    public BloatedRelation withMembers(final RelationBean members, final Rectangle bounds)
+    /**
+     * @param source
+     *            The relation that was used as a base to create that BloatedRelation, if any. Due
+     *            to the weak nature of relation membership across Atlas(es), this helps decide what
+     *            relation members are forcibly removed if any.
+     * @param members
+     *            The members of the relation
+     * @param bounds
+     *            The bounds of all the members of the relation.
+     * @return This.
+     */
+    public BloatedRelation withMembers(final Optional<Relation> source, final RelationBean members,
+            final Rectangle bounds)
     {
         this.members = members;
+        if (source.isPresent())
+        {
+            // This has been created from an existing relation, make sure to record the members that
+            // have been intentionally omitted, so as not to add them back in the future when either
+            // merging FeatureChanges or stitching MultiAtlases.
+            for (final RelationMember member : source.get().members())
```

Comment:
We probably want to do a `this.members.clear()` right above this `for` loop. That way, if someone does:
```java
bloatedRelation.withMembers(newMembers)
.
. // more code here
.
bloatedRelation.withMembers(someConditionalDictatesToUseTheseInstead)
```
you don't end up with a corrupted `members` set.

Reply:
Ah you're right. I missed that line :)